src/collections/_documentation/data-management/rollups.md

```diff
@@ -1,21 +1,48 @@
 ---
-title: 'Rollups & Grouping'
+title: 'Grouping & Fingerprints'
 sidebar_order: 0
 ---
 
-An important part of Sentry is how it aggregates similar events together and creates rollups. This turns out to be a pretty complex issue and it can be confusing for users about why some information might not be grouped correctly.
+An important part of Sentry is how similar events are grouped together. This turns out to be a pretty complex issue and it can be confusing for users about why some information might not be grouped correctly.
 
-The structured data of an event dictates how a rollup is created. This varies depending on the data available as well as the language.
+Errors are aggregated (and deduplicated) based on the fingerprint of the event. By default Sentry will generate its own fingerprint, but you can also customize this behavior using `scope.set_fingerprint` or equivilent API in our SDKs. For example, in JavaScript this might look like:
 
-## Grouping Priorities
+```javascript
+function makeRequest(method, path, options) {
+    return fetch(method, path, options).catch(err => {
+        Sentry.withScope(scope => {
+          // extend the fingerprint with the request path
+          scope.setFingerprint(['{{ default }}', path]);
+          Sentry.captureException(err);
+        });
+    });
+}
+```
 
-The grouping switches behavior based on the interfaces ([_Interfaces_]({%- link _documentation/development/sdk-dev/interfaces.md -%})) that are available within an event.
+The default fingerprint is generated based on information available to Sentry within the event (referred to as [_interfaces_]({%- link _documentation/development/sdk-dev/interfaces.md -%})). Fingerprints prioritize higher-value and more precise data when possible.
+
+A simple way to understand the logic is:
 
 -   If the interfaces used in an event differ, then those events will not be grouped together.
 -   If a stack trace or exception is involved in a report, then grouping will only consider this information.
--   If a template is involved, then grouping will consider the template.
+-   (Uncommon) If a template is involved, then grouping will consider the template.
```

Comment:
Uncommon enough that I'm realizing I don't actually have any idea what this means. What kind of a template? From where?

Is it something where it would be possible to add something succinct yet helpfully explanatory?